tests/unit/CMakeLists.txt

```diff
@@ -634,8 +635,20 @@ add_custom_target(tls_files
   DEPENDS ${out_tls_certificate_files}
 )
 
+if (NOT Seastar_TESTING_WITH_NETWORKING)
```

Comment:
Would it not make sense to always include the !TESTING_WITH_NETWORKING and just filter out outcall tests?
I can't imagine they add much to run time, and more testing by default is good.

Reply:
i don't feel strong either way. just wanted to have a decent coverage with relatively small efforts. will always enable the local tests and disable the tests using network access with the new option.